aya/src/util.rs

```diff
@@ -381,6 +400,17 @@ mod tests {
 
     #[test]
     fn test_parse_kernel_version_string() {
+        // cat /proc/version_signature on Proxmox VE 8.1.4
+        assert_matches!(
+            KernelVersion::parse_ubuntu_kernel_version(""),
+            Err(CurrentKernelVersionError::ParseError(s)) => { s.is_empty() }
```

Comment:
I think here you mean `if s.is_empty() => {}`?

Reply:
What's the deal with this `=> { ... }` syntax in `assert_matches!` anyway? I don't see it mentioned in the [documentation](https://doc.rust-lang.org/stable/std/assert_matches/macro.assert_matches.html), and the [tracking issue](https://github.com/rust-lang/rust/issues/82775) has decided against it.

Reply:
We are using [this crate](https://crates.io/crates/assert_matches).

Reply:
Oh. Anyway, it should just be `Err(CurrentKernelVersionError::ParseError(s)) if s.is_empty()`.